surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,152 @@
+# Node.js Promise reject use case survey
+
+Today, Node.js handles unhandled rejections by emitting a deprecation warning to stderr. The warning shows the stack where the rejection happened, and states that in future Node.js versions unhandled rejections will result on Node.js exitting with non-zero status code. We intend to remove the deprecation warning, replacing it with a stable behavior which might be different from the one described on the deprecation warning. We're running this survey to better understand how Node.js users are using Promises and how they are dealing with unhandled rejections today, so we can make an informed decision on how to move forward.
+
+## What is a Promise rejection?
+
+A Promise rejection indicates that something went wrong while executing a Promise or an async function. Rejections can occur on several situations: thorwing inside an async function or a `Promise` executor/then/catch/finally callback, when calling the `reject` callback of an `executor`, or when calling `Promise.reject`.
+
+```js
+
+Promise.reject()  // This will result in a rejection
```

Comment:
I mean, I know it's a bit silly but I would prefer if all these examples show `.reject(new Error())` just to not show people code that rejects with undefined without a stack trace 😅